### DISCUSSION TITLE
Install .qpkg over app store

### DISCUSSION THREAD

**Discussion by RJDavison** [pdulvp/jellyfin-qnap#93] [General]
Would there be any potential issue of manually installing the latest .qpkg package over the version of Jellyfin that was installed from your Appstore?

**Comment by NanashiTheNameless** [comment]
not that im aware of, but im not affiliated so grain of salt
(ive did it a few times and havent noticed anyting)

**Comment by pdulvp** [comment]
Hi, no issues, 
the store directly reference the qpkg from github.

**Reply by RJDavison** [reply]
Yep, all working. I did install the version with ffmpg 5 as is running an NVIDIA GPU. Would there be an issue installing the package with ffmpg 6 over the top of I ever decide too. Would it leave orphaned files or it's all running in a sandbox?

**Reply by pdulvp** [reply]
no issues no.
when installing a new version, it shall remove all previously installed files and leave no orphaned files.
